test/runtests.jl

```diff
@@ -503,6 +510,20 @@ LinearAlgebra.factorize(L::Chebyshev) =
 # This is wrong but just for tests
 Base.broadcasted(::LazyQuasiArrayStyle{2}, ::typeof(*), a::Expansion{<:Any,<:Chebyshev}, b::Chebyshev) = b * Matrix(I, 5, 5)
 
+struct QuadraticMap{T} <: Map{T} end
```

Comment:
@TSGut This now works. Note the simpler construction: just need to overload `invmap`, `getindex`, `axes`. 

I'll merge and tag this soon.